src/utils/actions.js

```diff
@@ -301,119 +309,44 @@ const actions = [
     translation: 'Object.ActionButton.ExecuteAutomation',
     value: 'executeAutomation',
     getActionCall:
-      ({ app, automation, automationId, automationTriggered, automationExecutionToken, automationPostData, buttonId, multiUserAutomation }) =>
-        async () => {
-          if (multiUserAutomation) {
-            let automationUrl
-            if (automation !== undefined && automationId.length < 1) {
-              const a = await getAutomationFromItem(automation);
-              automationUrl = `../api/v1/automations/${a.id}/runs`;
-            }
-            else if (automationId !== undefined && automationTriggered) {
-              automationUrl = `../api/v1/automations/${automationId}/actions/execute?X-Execution-Token=${automationExecutionToken}`;
-            }
-            else if (automationId !== undefined && !automationTriggered) {
-              automationUrl = `../api/v1/automations/${automationId}/runs`;
-            }
-            else {
-              return;
-            }
-            let bookmark
-            if (automationPostData) {
-              const bookmarkProps = {
-                qOptions: {
-                  qIncludeVariables: true,
-                  qIncludeAllPatches: true
-                }
-              };
-              bookmark = await app.createTemporaryBookmark(bookmarkProps);
-            }
-            const user = await getUser();
-            const inputs = {
-              app: app.id,
-              bookmark: automationPostData ? bookmark : '',
-              sheet: await getSheetId(app, buttonId),
-              user: user.subject,
-              space: await getSpaceId(app.id),
-              tenant: user.tenantId,
-              time: new Date(),
-            }
-            const automationData = {
-              id: automationId,
-              inputs,
-              context: 'qsbutton'
-            }
-            const headers = {
-              'Content-Type': 'application/json',
-              'qlik-csrf-token': await getCsrfToken()
-            }
-            if (automationTriggered) {
-              headers['X-Execution-Token'] = automationExecutionToken;
-            }
-            const postOptions = {
-              method: 'POST',
-              headers,
-              body: JSON.stringify(automationTriggered ? inputs : automationData),
-            }
-            await fetch(automationUrl, postOptions);
+      ({
+        app,
+        automation,
+        automationId,
+        automationTriggered,
+        automationExecutionToken,
+        automationPostData,
+        automationShowNotification,
+        automationNotificationDuration,
+        buttonId,
+        automationOpenLinkInNewTab,
+        multiUserAutomation,
+        translator
+      }) =>
+      async () => {
+        if (multiUserAutomation && automationId.length ) {
```

Comment:
There is a lot of backend calls inside this if statement. You probably want to do the same as in the old handling and put everything inside a catch function